src/ebuckets.c

```diff
@@ -1876,6 +1876,90 @@ uint64_t ebGetExpireTime(EbucketsType *type, eItem item) {
     return ebGetMetaExpTime(meta);
 }
 
+/* Init ebuckets iterator
+ *
+ * This is a non-safe iterator. Any modification to ebuckets will invalidate the
+ * iterator. Calling this function takes care to reference the first item
+ * in ebuckets with minimal expiration time. If not items to iterate, then
+ * iter->currItem will be NULL */
+void ebStartIter(EbucketsIterator *iter, ebuckets eb, EbucketsType *type) {
```

Comment:
should we consistent with the other iterators(raxStart, dictGetIterator), start does not fetch the first element, but the first next calling.

Reply:
Rax iterator is rather heavy weight. Even after `raxStart()`, we need to call `raxSeek()`. And only then `raxNext()`. 
IMHO, given that aligning such an API might not be ideal, I felt the freedom to keep it as simple as possible.

Reply:
but from the user's perspect, would it be easier to write this way?

```
    ebStartIter(&iter, eb, &myEbucketsType);
    while (ebNext(&iter)) {
		// bla bla
    }
    ebStopIter(&iter);
```

Reply:
In C++, the iterator on begin() already points to the first item:
```
   vector<int> ar = { 1, 2, 3, 4, 5 }; 
    vector<int>::iterator ptr;       
    for (ptr = ar.begin(); ptr < ar.end(); ptr++) ...
```